src/renderers/webgpu/WebGPUBackend.js

```diff
@@ -64,7 +64,7 @@ class WebGPUBackend extends Backend {
 				powerPreference: parameters.powerPreference
 			};
 
-			const adapter = await navigator.gpu.requestAdapter( adapterOptions );
+			const adapter = typeof navigator === 'undefined' ? false : await navigator.gpu.requestAdapter( adapterOptions );
```

Comment:
Is the compiled application still usable if you use `false` here?

In any event, I would at least use `{}` as a fallback.

Reply:
Besides, please write the tests like so:
```js
const adapter = ( typeof navigator !== 'undefined' ) ? await navigator.gpu.requestAdapter( adapterOptions ) : {};
```
Meaning you test for the normal case first and then handle the fallback.

Reply:
Thanks, fixed.